android/java/org/chromium/chrome/browser/rewards/BraveRewardsPanel.java

```diff
@@ -650,15 +631,13 @@ private void showNotification(String id, int type, long timestamp, String[] args
         View claimProgress = mPopupView.findViewById(R.id.claim_progress);
         NotificationClickAction notificationClickAction = NotificationClickAction.DO_NOTHING;
 
-        mClaimInProcess = mBraveRewardsNativeWorker.isGrantClaimInProcess();
-        if (mClaimInProcess) {
-            BraveRewardsHelper.crossfade(actionNotificationButton, claimProgress, View.GONE, 1f,
-                    BraveRewardsHelper.CROSS_FADE_DURATION);
-        } else {
-            actionNotificationButton.setEnabled(true);
-            BraveRewardsHelper.crossfade(claimProgress, actionNotificationButton, View.GONE, 1f,
-                    BraveRewardsHelper.CROSS_FADE_DURATION);
-        }
+        actionNotificationButton.setEnabled(true);
```

Comment:
we can remove the change here. as it won't be needed. i will remove it in the follow up issue.

Reply:
I'll leave this for for a follow-up.